### PR TITLE
fix bugs

### PR DESCRIPTION
**Pull request by cdebourcy**
fix bad behavior in CSV generation and remove the download button for now

### CODE REVIEW

File: app/controllers/reports_controller.rb [hunk 1]
```diff
@@ -74,6 +74,16 @@ def destroy
   end
 
   def send_report_csv
+    # override DEFAULT_PARAMS of report_helper.rb to get raw data
```
Comment (yunfangjuan) [inline app/controllers/reports_controller.rb]: a bit hacky. probably should set a flag in geneate_report_csv to turn off the thresholdings. genus_search would need it too. @boris-dimitrov. At least at a TODO here.

File: app/lib/report_helper.rb [hunk 2]
```diff
@@ -563,10 +563,12 @@ def get_tax_detail(tax_info, column_name)
 
   def generate_report_csv(report_info)
     rows = report_info[:taxonomy_details][1]
+    view_level_int = TaxonCount::NAME_2_LEVEL[report_info[:report_page_params][:view_level].downcase]
     attributes = %w[category_name tax_id name NT.zscore NT.rpm NT.r NR.zscore NR.rpm NR.r]
     CSV.generate(headers: true) do |csv|
       csv << attributes
       rows.each do |tax_info|
+        next unless tax_info['tax_level'] == view_level_int
```
Comment (boris-dimitrov) [inline app/lib/report_helper.rb]: This removes genus entries from species results.   That's what you want?

File: db/schema.rb [hunk 3]
```diff
@@ -94,7 +94,7 @@
   end
 
   create_table "projects", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
-    t.string "name", collation: "latin1_swedish_ci"
+    t.string "name", collation: "utf8_general_ci"
```
Comment (boris-dimitrov) [inline db/schema.rb]: remove this file from pull request?

File: app/controllers/reports_controller.rb [hunk 4]
```diff
@@ -74,6 +74,16 @@ def destroy
...
+    params[:threshold_zscore] = 0
+    params[:threshold_rpm] = 0
+    params[:threshold_r] = 0
+    params[:threshold_percentidentity] = 0
+    params[:threshold_alignmentlength] = 0
+    params[:threshold_neglogevalue] = 0
+    params[:threshold_aggregatescore] = 0
+    params[:excluded_categories] = 'None'
+    # serve CSV
```
[the remark this answers is not in the document]
Comment (cdebourcy) [reply app/controllers/reports_controller.rb]: Ok

File: app/controllers/reports_controller.rb (hunk 1 above)
Comment (cdebourcy) [reply app/controllers/reports_controller.rb]: ok

File: app/lib/report_helper.rb (hunk 2 above)
Comment (cdebourcy) [reply app/lib/report_helper.rb]: yes

**Pull request merged**